features/administration.feature

```diff
@@ -59,3 +59,32 @@ Feature: Bookstore Administration Panel
     Then I see the book "Book That Has Been Edited"
     When I visit the public book index
     Then I see the book "Book That Has Been Edited"
+
+  Scenario: Viewing orders
+    Given I am logged into the admin panel
+    And there are 100 books in the database
+    And some books have been ordered
+    When I visit the admin orders url
+    Then I see the orders
+
+  Scenario: Adding a book discount
+    Given I am logged into the admin panel
+    And I am logged into the site
+    And there is a book named "Book To Be Discounted"
+    And the book is valued at "$20.00"
```

Comment:
I'm guessing this was given to you, but a better way to do this would probably be:

````gherkin
And the database has the following book:
      | name | price |
      | Book To Be Discounted | $20.00 |
````